CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to
 
 ### Added
 
+- dev: bump reth
```

Comment:
probably doesn't deserve its changelog entry;p

Reply:
Gotcha, let me remove this and add `skip-changelog` label 🫡